---
bool success
coverage_control/Grid map